src/scripts-upload-media/factcheck.js

```diff
@@ -0,0 +1,97 @@
+const axios = require("axios");
+const path = require("path");
+const { v4: uuid } = require("uuid");
+require("dotenv").config({
+    path: path.join(__dirname, "development.env"),
+});
+const fs = require("fs")
+
+const accessToken = process.env.ACCESS_TOKEN;
+const datasource = "60212d4c-e870-46a6-907b-88ef39e0b88f";
+const knownMediaTypes = ['text', 'image', 'video'];
+const ignoreMediaTypes = ['tweet', 'facebook', 'instagram'];
+const failedRequestsFile = "factcheck-failed-requests.json";
+
+const insertPosts = async () => {
+    const { connect, get, bulkWrite } = require("../backend/core/mongo/index")
+    await connect()
+
+    const stories = await get(
+        "kosh_metadata",
+        "stories",
+        { "docs.e_kosh_id": { $exists: false }, "docs.mediaType": { $nin: ignoreMediaTypes } },
+        { limit: 100, skip: 0 }
+    )
+    const result = {totalPosts: 0, failed: []}
+    const posts = []
+    const unknownMediaTypes = new Set()
+    for await (const story of stories) {
+        if (!story.docs || !Array.isArray(story.docs)) {
+            continue
+        }
+        for (const post of story.docs) {
+            if (post.e_kosh_id) continue
+            if (!knownMediaTypes.includes(post.mediaType)) {
+                unknownMediaTypes.add(post.mediaType)
+                continue
+            }
+            if (post.mediaType === "text" && !post.s3URL) {
+                const fileId = uuid();
+                const { uploadData } = require("./s3");
+                const awsResponse = await uploadData(post.content, fileId);
+                post.s3URL = awsResponse.Location
```

Comment:
replace this with post.s3uRL = `https://fs.tattle.co.in/service/kosh/file/${fileId}`
We'd not want to expose the raw s3 url anywhere on the website. I should have flagged this earlier. Sorry about that.